src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -13618,21 +13632,42 @@ void Player_Action_8084FBF4(Player* this, PlayState* play) {
     func_8002F8F0(&this->actor, NA_SE_VO_LI_TAKEN_AWAY - SFX_FLAG + this->ageProperties->unk_92);
 }
 
-s32 func_8084FCAC(Player* this, PlayState* play) {
+#if OOT_DEBUG
+/**
+ * Updates the "Noclip" debug feature, which allows the player to fly around anywhere
+ * in the world and clip through any collision.
+ *
+ * Noclip can be enabled with two different button combos:
+ * Hold A + L + R and press B
```

Comment:
the
`if (!CHECK_BTN_ALL(sControlInput->cur.button, BTN_L))`
check before checking for A/B
makes me believe their combo was holding L before A, so Link isn't moved by holding A
anyway even for just practical reasons I'd suggest
```suggestion
 * Hold L + R + A and press B
```
the order in `BTN_A | BTN_L | BTN_R` is completely arbitrary after all (in this case from most significant bit to least)

Reply:
sure, done

Reply:
I also changed the button check to reflect this